Asterix/Example_param_file.ini

```diff
@@ -278,7 +279,8 @@ onbench=True
     
     # Dark Hole Mask parameters
 
-    # "circle", "square", "noDH" or "custom"
+    # "circle", "square", "nodh" or "custom" (if custom, letters are harcoded in creatingMaskDH function in maskDH.py)
+    # Can also be a fits name with a mask of the right size (dimEstimxdimEstim)
```

Comment:
Please put spaces around the `x`, the current thing is fairly unreadable

Reply:
done